blob/azureblob/example_test.go

```diff
@@ -25,74 +26,171 @@ import (
 
 func Example() {
 	ctx := context.Background()
-
 	accountName, accountKey := "", ""
```

Comment:
What happens if you provide invalid credentials instead of empty ones? It would be nice (because more of the code would actually run) if you got the error from `ReadAll` instead of from `ServiceURLFromAccountKey`, similar to the GCS example.

Reply:
Using invalid account/keys will result in a fail test due to a timeout exception. The Azure download operation (for ReadAll) will fail after 30 seconds with a HTTP timeout exception. 30 seconds represents the default timeout on a blob operation.

panic: test timed out after 30s
https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-blob-service-operations

Reply:
I can fix this with a custom retry policy on the pipeline to ensure the calls return under 30 seconds. Let me know if this is OK.

opts := azblob.PipelineOptions{
		Retry : azblob.RetryOptions{	
			Policy: azblob.RetryPolicyFixed,
			TryTimeout : 2 * time.Second,		
			MaxTries : 1,
			RetryDelay : 0 * time.Second,
			MaxRetryDelay : 0 * time.Second,
		},
	}
	**p := azblob.NewPipeline(creds, opts)
	*serviceURL = serviceURL.WithPipeline(p)**